### DISCUSSION TITLE
Handle http/https connections

### DISCUSSION THREAD

**Discussion by ahmad939** [sumatrapdfreader/sumatrapdf#3191] [Ideas]
It seems that Sumatra PDF Reader can't handle pdf file downloads over http or https protocols. I still get errors no matter which way I try.
This is very useful functionality when surfing online.

**Comment by GitHubRulesOK** [comment]
unsure what you mean I often open https: from SumatraPDF, the problem is its very slow since its channelled by windows and thus legacy Trident Browser. It works BUT Its often faster to curl and load a remote URL using batch files.
![image](https://user-images.githubusercontent.com/12754913/209439667-bf41ef7a-5d67-4613-90d1-adaa1d8f8d16.png)
![image](https://user-images.githubusercontent.com/12754913/209439797-ab7d44a4-bf2e-4c68-8d1c-877191348758.png)
![image](https://user-images.githubusercontent.com/12754913/209440596-2ab8a98a-8f67-47ad-bdb3-07b985bda31e.png)
![image](https://user-images.githubusercontent.com/12754913/209441002-21d0ccd3-457c-4439-9424-cf1e29ed644f.png)
